### PR TITLE
Added logo to navbar

### PR DESCRIPTION
**Pull request by jbpratt78**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import cs from 'classnames';
 import idx from 'idx';
-
+import logo from '../../public/image/ComfyApe.png'
```
Comment (KenanY) [inline src/components/Header.jsx]: This seems to add ~4.71 KB (gzipped) to the main JS file:

image

could we instead do something like ``?

File: src/components/Header.jsx [hunk 2]
```diff
@@ -69,7 +69,9 @@ const Header = ({
       role='navigation'
     >
       <div className='navbar-header'>
-        <Link className='navbar-brand' to='/'>Strims</Link>
+        <Link className='navbar-brand' to='/'>
+          <img src={logo} />
```
Comment (KenanY) [inline src/components/Header.jsx]: consider adding an `alt` tag with value `'Strims'` (for screen readers or for when the image fails to load)

File: src/components/Header.jsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ const Header = ({
     >
       <div className='navbar-header'>
         <Link className='navbar-brand' to='/'>
-          <img src='/image/comfyape.png' alt='Strims' />
+          <img src='/image/comfyape.png' alt='Strims' /> Strims
```
[the remark this answers is not in the document]
Comment (jbpratt78) [reply src/components/Header.jsx]: tfw you leave a review on pending for 5 days

**Comment by xDashh** [comment]
Reminder to take this into account https://i.imgur.com/h4gsuqm.png

**Comment by jbpratt78** [comment]
Closing, opening new PR when the new navbar has been merged.

**Pull request closed without merging**